### ISSUE TITLE
Prevent serverHostname from being 0.0.0.0

### ISSUE THREAD

**Issue by paleite** [docpad/docpad#911]
Chrome has recently stopped support for using http://0.0.0.0 as a valid address and if you input http://0.0.0.0 into the omnibar in Chrome version >40 you'll be redirected to a search instead.

The issue is presented here:
https://chromium.googlesource.com/chromium/src/+/adab4b9162250dae2fb858029d03fc39c6ae9393

And it's also discussed (although in regards to Jekyll) here:
https://code.google.com/p/chromium/issues/detail?id=428046

Which in turn references this URL:
https://github.com/jekyll/jekyll/pull/3053

**Comment by Greduan** [comment]
Why is there no label for critical? lol @mikeumus 

/cc @balupton

**Comment by balupton** [comment]
There is a different between serving and requesting on 0.0.0.0. Serving on 0.0.0.0 will make the website accessible on all ip addresses (your local, your remote, and your loop-back). Perhaps we should still serve on 0.0.0.0 but alter the display of the URL in the console to 127.0.0.1

**Comment by StephenBrown2** [comment]
:+1:  I second this. Even changing to 'localhost' would make sense.

**Comment by balupton** [comment]
Will implement my suggestion above in the next release.

**Comment by balupton** [comment]
Added in v6.70.1

**Comment by StephenBrown2** [comment]
Hate to re-open this, but this appears to have reverted to IPv6 with an update to node 12?

```
% node --version
v0.12.0
% docpad run
info: Welcome to DocPad v6.70.1 (local installation: /home/.../docpad)
notice: If everyone who saw this message donated $1/week, maintaining DocPad would become sustainable: http://docpad.org/donate
info: Contribute: http://docpad.org/docs/contribute
info: Plugins: associatedfiles, coffeemugg, copy, data, eco, ghpages, imagin, livereload, marked
info: Environment: development
info: DocPad listening to http://:::9778/ on directory /home/gilligan/scottroyston.docpad/out
info: LiveReload listening to new socket on channel /docpad-livereload
...
```

**Comment by balupton** [comment]
Hahahaha. Ok will fix.

**Comment by balupton** [comment]
Fixed in v6.71.1
